task

```diff
@@ -1066,6 +1101,19 @@ function update-docs() {
             done
         done
     done
+
+    generate-revisions
+    if ! grep -q "\*\*Odoo Revisions\*\*" README.md; then
+        echo "Odoo Revisions section not found in README.md."
+        exit 1
+    fi
+
+    if ! grep -q "\*\*\[Odoo Revisions\](revisions.md)\*\*" README.md; then
+        echo "Linking revisions.md in README.md"
+        sed -i 's/\*\*Odoo Revisions\*\*/\*\*\[Odoo Revisions\](revisions.md)\*\*/' README.md
+    else
+        echo "Link to revisions.md already exists in README.md"
+    fi
     echo -e "export default [$SIDEBAR_ENTRIES]" > .vuepress/sidebar.js
 }
 
```

Comment:
The part where the function actually is called is missing.

Reply:
It is called above the update of the README